static/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.tsx

```diff
@@ -277,6 +282,90 @@ export default class SentryApplicationDetails extends AsyncView<Props, State> {
     }
   };
 
+  getAvatarPreview = (avatarStyle: 'color' | 'simple') => {
+    const {app} = this.state;
+    const styleMap = {
```

Comment:
For dict-like variables like this I prefer to make it a const at the top level of the file.
```tsx
export const AVATAR_STYLES = {
  color: {
    size: 50,
    title: t('Default Logo'),
    description: t('The default icon for integrations'),
  },
  simple: {
    size: 20,
    title: t('Default Icon'),
    description: t('This is an optional icon used for Issue Linking'),
  },
};
```